metadatastore/commands.py

```diff
@@ -336,60 +337,39 @@ def find_run_start(limit=50, **kwargs):
     ------
     >>> find_run_start(scan_id=123)
     >>> find_run_start(owner='arkilic')
-    >>> find_run_start(time={'start': 1421176750.514707,
-    ...                      'stop': time.time()})
-    >>> find_run_start(time={'start': 1421176750.514707,
-    ...                      'stop': time.time()})
+    >>> find_run_start(start_time=1421176750.514707, stop_time=time.time()})
+    >>> find_run_start(start_time=1421176750.514707, stop_time=time.time())
 
-    >>> find_run_start(owner='arkilic', time={'start': 1421176750.514707,
-    ...                                       'stop': time.time()})
+    >>> find_run_start(owner='arkilic', start_time=1421176750.514707,
+    ...                stop_time=time.time())
 
     """
-    search_dict = dict()
-
-    try:
-        search_dict['scan_id'] = kwargs.pop('scan_id')
-    except KeyError:
-        pass
-
-    try:
-        search_dict['unique_id'] = kwargs.pop('unique_id')
-    except KeyError:
-        pass
-
-    try:
-        search_dict['owner'] = kwargs.pop('owner')
-    except KeyError:
-        pass
-
+    # format time correctly
     try:
-        search_dict['beamline_id'] = kwargs.pop('beamline_id')
+        start_time = kwargs['start_time']
     except KeyError:
         pass
+    else:
+        kwargs['time'] = {'$gte': start_time}
 
     try:
-        time_dict = kwargs.pop('time')
-        if not isinstance(time_dict, dict):
-            raise ValueError('Wrong format. time must include '
-                             'start and stop keys for range. Must be a dict')
-        else:
-            if 'start' in time_dict and 'stop' in time_dict:
-                search_dict['time'] = {'$gte': time_dict['start'],
-                                       '$lte': time_dict['stop']}
-            else:
-                raise ValueError('time must include start '
-                                 'and stop keys for range search')
+        stop_time = kwargs['stop_time']
```

Comment:
It might be a bit less performant, but I think something like

```python
time_dict = {}
start_time = kwargs.pop('start_time', None)
stop_time = kwargs.pop('stop_time', None)
if start_time:
    time_dict['$gte'] = start_time
if stop_time:
    time_dict['$lte'] = stop_time
if time_dict:
    kwargs['time'] = time_dict
```

Would be clearer